cdn/signedurls/signurl_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func Test_readKeyFile(t *testing.T) {
+	f, err := ioutil.TempFile("", "cdnkey")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(f.Name())
+	f.Close()
+
+	key := `nZtRohdNF9m3cKM24IcK4w==`
```

Comment:
Might be prudent to put this key in a file in testdata, so you don't have to mess around with TempFile.

But then that introduces a testdata directory.

Reply:
yeah I think inline is fine. this one is testing the file read functionality anyway.

Reply:
yeah probably a better idea to keep it as is.